### PR TITLE
Theme arc-green: reverse heatmap colors

### PR DESCRIPTION
**Pull request by cdlm**
This uses the same colors as the updated palette in the base theme.

See #8709 and #5864, in particular [my comment showing the problem](https://github.com/go-gitea/gitea/issues/5864#issuecomment-462334171)

### CODE REVIEW

**Comment by jaqra** [comment]
@cdlm thanks for pr. can you add screenshots for white and green themes?

**Comment by cdlm** [comment]
I didn't touch the white theme, so @MayMeow's [screenshot](https://github.com/go-gitea/gitea/pull/8709#issuecomment-546733804) should be good.

Here's a screenshot of my gitea instance with the proposed change:
updated arc-green heatmap palette

Now that I see it in context, the gradient looks a little too saturated, with the middle green even a little out of place… Maybe we can iterate on the precise palette, but at least the gradient goes in the right direction.

**Review by ghost — APPROVED** [review]

**Comment by silverwind** [comment]
I think colors should fade towards the background color with decreasing activity, not towards white like you have. Can you do something similar like here?

**Review by sapk — APPROVED** [review]
This could be improved by using some ~~rgba()~~ other gradient like suggested by @silverwind in https://github.com/go-gitea/gitea/pull/8840#issuecomment-550249723 but this is already an improvement.

**Comment by cdlm** [comment]
@silverwind I agree; in fact I had a version which was using the original colors from arc-green (from the default theme actually, since arc-green was only redefining the "no contribution" color) and it was a bit nicer.

**Comment by cdlm** [comment]
Sorry about the force-push… I rebased my changes on top of master, merging the CSS-only commits into one. Here's what it looks with color interpolation to the same green as text links:
Screenshot 2019-11-06 at 15 30 59

BTW my activity on this instance is… sparse and with few commits, which is why only some colors of the range appear in the grid.

File: public/less/themes/arc-green.less [hunk 1]
```diff
@@ -1294,8 +1294,34 @@ a.ui.labels .label:hover {
     }
 }
 
+.heatmap(@heat) {
+    @heatmap-cold: #2d303b;
+    @heatmap-hot: #87ab63;
```
Comment (sapk) [inline public/less/themes/arc-green.less]: Maybe we can use the color for a:hover (#a0cc75) which is a little brighter ? I feel like the heatmap is too dark now 😄

Comment (jaqra) [reply public/less/themes/arc-green.less]: And how #00FF48 looks? (i can't imagine)

Comment (cdlm) [reply public/less/themes/arc-green.less]: Good point 👍

Comment (cdlm) [reply public/less/themes/arc-green.less]: With `#a0cc75` from `a:hover` (I like it):
Screenshot 2019-11-06 at 16 08 59

With `#00ff48` it looks out of place IMHO… too vibrant compared to the rest of the theme:
Screenshot 2019-11-06 at 16 12 08

**Comment by sapk** [comment]
The CI failure seems unrelated. I restarted it.

**Review by sapk — APPROVED** [review]

**Pull request merged**